Regression in `@fontsource/poppins` v5.2.0 – Imports for Specific Weights and Styles Fail

### Describe the bug

After updating `@fontsource/poppins` from v5.1.1 to v5.2.0, importing specific font weights and styles in our React application now results in build errors.

Previously, we imported fonts as follows without issues:

```tsx
import '@fontsource/poppins/200.css';
import '@fontsource/poppins/300.css';
import '@fontsource/poppins/400-italic.css';
import '@fontsource/poppins/400.css';
import '@fontsource/poppins/500.css';
import '@fontsource/poppins/600.css';
import '@fontsource/poppins/700.css';
```

Upon upgrading to v5.2.0, we now receive the following Webpack errors:

```
ERROR in ./src/index.tsx 1:0-37
Module not found: Error: Package path ./200.css is not exported from package @fontsource/poppins (see exports field in package.json)
```

This occurs for all weight/style imports.

### **Observed Changes in `package.json`**
The `exports` field in v5.2.0 now restricts what is accessible:

```json
"exports": {
  ".": { "sass": "./index.css", "default": "./index.css" },
  "./scss": { "sass": "./scss/metadata.scss" }
}
```

This appears to prevent direct imports of individual weight/style files.

### **Environment Details**
- Node.js version: 22.14.0
- Package manager npm and version: 11.1.0
- Build system: Webpack 5
- OS: macOS Sequoia 15.3.1

### Steps to Reproduce

Imported fonts:

```tsx
import '@fontsource/poppins/200.css';
import '@fontsource/poppins/300.css';
import '@fontsource/poppins/400-italic.css';
import '@fontsource/poppins/400.css';
import '@fontsource/poppins/500.css';
import '@fontsource/poppins/600.css';
import '@fontsource/poppins/700.css';
```

### **Steps to Reproduce**
1. Install `@fontsource/poppins@5.2.0`
2. Import fonts as shown above
3. Run `npm run build`
4. Observe Webpack errors related to missing exports

### Expected behavior

- Importing weights and styles as separate `.css` files should continue to work as in previous versions.

### Version

5.2.0

### OS

macOS Sequoia 15.3.1

### Browser

_No response_

### Additional context

### **Workarounds**
For now, we have downgraded to v5.1.1 to restore functionality.

Could you confirm if this change was intentional, or if there's an alternative way to import font weights and styles in v5.2.0?

Thanks for your help!

--- a/.changeset/tidy-coats-love.md
+++ b/.changeset/tidy-coats-love.md
@@ -1,0 +1,5 @@
+---
+"@fontsource-utils/cli": patch
+---
+
+fix(template): widen exports field to support all CSS imports

--- a/packages/cli/src/templates/package.ts
+++ b/packages/cli/src/templates/package.ts
@@ -39,6 +39,17 @@ const template = (
 			sass: './index.css',
 			default: './index.css',
 		},
+		'./*': {
+			sass: './*.css',
+			default: './*.css',
+		},
+		'./files/*': {
+			sass: './files/*',
+			default: './files/*',
+		},
+		'./package.json': './package.json',
+		'./metadata.json': './metadata.json',
+		'./unicode.json': './unicode.json',
 		'./scss': {
 			sass: './scss/metadata.scss',
 		},
@@ -46,9 +57,10 @@ const template = (
 	author: type === 'other' ? license.attribution : 'Google Inc.',
 	license: license.type,
 	homepage: `https://fontsource.org/fonts/${id}`,
+	funding: 'https://github.com/sponsors/ayuhito',
 	repository: {
 		type: 'git',
-		url: 'https://github.com/fontsource/font-files.git',
+		url: 'git+https://github.com/fontsource/font-files.git',
 		directory: `fonts/${isVariable ? 'variable' : type}/${id}`,
 	},
 	// If publish hash exists, add it to the package.json

